webapp/javascript/index.jsx

```diff
@@ -30,7 +29,7 @@ try {
 
 ReactDOM.render(
   <Provider store={store}>
-    <BrowserRouter history={history} basename={basename()}>
```

Comment:
@petethepig wasn't there a reason you felt you needed `BrowserRouter` instead of `Router` ?